docs/user_guide/use_measured_boot.rst

```diff
@@ -10,22 +10,22 @@ Introduction
 ------------
 
 In any real-world large-scale production environment, a large number of
-different types of nodes will typically be found. The TPM 2.0 defines specific
-meaning - measurement of UEFI bios, measurement of boot device firmware - for
-lower-numbered PCRs (e.g., PCRs 0-9), as these are extended during the multiple
-events of a measured boot log. However, simply comparing the contents of these
-PCRs against a well-known "golden value" becomes unfeasible. The reason for
-this is, in addition to the potentially hundreds of variations due to node
-type, that it can be experimentally demonstrated that some PCRs (e.g, PCR 1)
-vary for each physical machine, if such machine is netbooted (as it encodes the
-MAC address of the NIC used during boot.)
+different types of nodes will typically be found. The TPM 2.0 defines a
+specific meaning - measurement of UEFI bios, measurement of boot device
+firmware - for each of the lower-numbered PCRs (e.g., PCRs 0-9), as these are
+extended during the multiple events of a measured boot log. However, simply
+comparing the contents of these PCRs against a well-known "golden value"
+becomes unfeasible. The reason for this is, in addition to the potentially
+hundreds of variations due to node type, it can be experimentally demonstrated
+that some PCRs (e.g, PCR 1) vary for each physical machine, if such machine is
+netbooted (as it encodes the MAC address of the NIC used during boot.)
 
 Fortunately, the UEFI firmware is now exposing the event log through an ACPI
 table and a "recent enough" Linux kernel (e.g., 5.4 or later) is now consuming
 this table and exposing this boot event log through the securityfs, typically
 at the path `/sys/kernel/security/tpm0/binary_bios_measurements`. When combined
-with `secure boot` and a "recent enough" version of grub (TBD), the
-aforementioned can be fully populated, including measurements of all
+with `secure boot` and a "recent enough" version of grub (2.06 or later), the
```

Comment:
I can be wrong, but 2.05 also measured via grub-tpm the kerne, initrl, grub.cfg and the kernel command line.

Reply:
Couldn't find evidence of it on 2.05, will keep 2.06 for now, but we can change later.